### PR TITLE
feat(vital-list): Add new vital list component

### LINKED ISSUE #1559: Error on a Edit env of starter site page with relative URLs /styleguide/, /___templates/styleguide/ "Warning: validateDOMNesting(...): <ul> cannot appear as a descendant of <p>."
STR:

Run Edit ENV of starter site and go to relative URL "/styleguide/"
Check console errors
ER: No errors are presented

AR: Next error is presented:

<code>
Warning: validateDOMNesting(...): <ul> cannot appear as a descendant of <p>.
ul
p
Stylable@http://localhost:8005/component---src-templates-styleguide-jsx.js:97690:25
AddClasses@http://localhost:8005/component---src-templates-styleguide-jsx.js:97571:31
AddClasses
Examples
ReplaceWith
WithMeta
WithMeta
WithNodeKey
WithMeta
WithMeta
WithMeta
WithMeta
Examples
section
Stylable@http://localhost:8005/component---src-templates-styleguide-jsx.js:97690:25
Replaceable@http://localhost:8005/component---src-templates-styleguide-jsx.js:98699:74
AddPropsIf@http://localhost:8005/component---src-templates-styleguide-jsx.js:97854:27
WithMeta
ExamplesWrapper
div
Stylable@http://localhost:8005/component---src-templates-styleguide-jsx.js:97690:25
Replaceable@http://localhost:8005/component---src-templates-styleguide-jsx.js:98699:74
AddPropsIf@http://localhost:8005/component---src-templates-styleguide-jsx.js:97854:27
AddClasses@http://localhost:8005/component---src-templates-styleguide-jsx.js:97571:31
AddClasses
AddClasses@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
AddClasses
AddClasses
AddClasses
AddClasses@http://localhost:8005/component---src-templates-styleguide-jsx.js:97571:31
AddClasses
AddClasses@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
AddClasses
AddClasses
AddClasses
AddClasses
AddClasses
AddClasses
AddClasses
Container
div
Stylable@http://localhost:8005/component---src-templates-styleguide-jsx.js:97690:25
Replaceable@http://localhost:8005/component---src-templates-styleguide-jsx.js:98699:74
AddPropsIf@http://localhost:8005/component---src-templates-styleguide-jsx.js:97854:27
AddPropsIf
WithMeta
WithMeta
WithMeta
Container
div
Stylable@http://localhost:8005/component---src-templates-styleguide-jsx.js:97690:25
Replaceable@http://localhost:8005/component---src-templates-styleguide-jsx.js:98699:74
AddPropsIf@http://localhost:8005/component---src-templates-styleguide-jsx.js:97854:27
AddClasses@http://localhost:8005/component---src-templates-styleguide-jsx.js:97571:31
AddClasses
AddClasses
AddClasses
ContainerWrapper
div
Stylable@http://localhost:8005/component---src-templates-styleguide-jsx.js:97690:25
Replaceable@http://localhost:8005/component---src-templates-styleguide-jsx.js:98699:74
AddPropsIf@http://localhost:8005/component---src-templates-styleguide-jsx.js:97854:27
AddClasses@http://localhost:8005/component---src-templates-styleguide-jsx.js:97571:31
AddClasses
AddClasses
AddClasses
OuterContainer
LayoutCleanBase@http://localhost:8005/component---src-templates-styleguide-jsx.js:90201:42
Transformer@http://localhost:8005/component---src-templates-styleguide-jsx.js:97381:34
WithDesign@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
extendDesignable(Layout)
Replaceable@http://localhost:8005/component---src-templates-styleguide-jsx.js:98699:74
AddPropsIf@http://localhost:8005/component---src-templates-styleguide-jsx.js:97854:27
StartWith@http://localhost:8005/component---src-templates-styleguide-jsx.js:98760:80
@http://localhost:8005/component---src-templates-styleguide-jsx.js:68392:68
WithMeta
WithDesign@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
WithMeta
WithMeta
WithMeta
WithDesign@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
WithMeta
WithMeta
WithMeta
WithDesign@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
WithMeta
WithMeta
WithMeta
WithDesign@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
WithMeta
WithMeta
WithMeta
WithDesign@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
WithMeta
WithMeta
WithMeta
WithDesign@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
WithMeta
WithMeta
WithMeta
WithMeta
WithMeta
WithMeta
WithMeta
Wrapper
StyleGuideTemplateBase@http://localhost:8005/component---src-templates-styleguide-jsx.js:96300:15
Transformer@http://localhost:8005/component---src-templates-styleguide-jsx.js:97381:34
WithDesign@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)@http://localhost:8005/component---src-templates-styleguide-jsx.js:98699:74
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
extendDesignable(StyleGuide)
Chameleon@http://localhost:8005/component---src-templates-styleguide-jsx.js:38815:106
WithoutProps
extendDesignable(StyleGuide)
useObserver@http://localhost:8005/component---src-templates-styleguide-jsx.js:119470:64
WithoutProps
WithMeta
useObserver@http://localhost:8005/component---src-templates-styleguide-jsx.js:119470:64
WithChameleonContext@http://localhost:8005/component---src-templates-styleguide-jsx.js:39879:40
EndSidecarNodes@http://localhost:8005/component---src-templates-styleguide-jsx.js:59295:69
useObserver@http://localhost:8005/component---src-templates-styleguide-jsx.js:119470:64
NodeProvider@http://localhost:8005/component---src-templates-styleguide-jsx.js:50255:18
WithNode@http://localhost:8005/component---src-templates-styleguide-jsx.js:58600:23
WithNodeKey
StartSidecarNodes@http://localhost:8005/component---src-templates-styleguide-jsx.js:59254:69
StartSidecarNodes
StartSidecarNodes
StartSidecarNodes
StartSidecarNodes@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
StartSidecarNodes
StartSidecarNodes
StartSidecarNodes
StartSidecarNodes@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
StartSidecarNodes
StartSidecarNodes
div
Stylable@http://localhost:8005/component---src-templates-styleguide-jsx.js:97690:25
WithoutProps
useObserver@http://localhost:8005/component---src-templates-styleguide-jsx.js:119470:64
ContextWrapper
useObserver@http://localhost:8005/component---src-templates-styleguide-jsx.js:119470:64
PageEditor
NotificationProvider@http://localhost:8005/component---src-templates-styleguide-jsx.js:50431:22
GitContextProvider@http://localhost:8005/component---src-templates-styleguide-jsx.js:48216:22
PageDataProvider@http://localhost:8005/component---src-templates-styleguide-jsx.js:74573:22
WithOnlyProps
@http://localhost:8005/component---src-templates-styleguide-jsx.js:99956:74
NodeProvider@http://localhost:8005/component---src-templates-styleguide-jsx.js:50255:18
NodeProvider@http://localhost:8005/component---src-templates-styleguide-jsx.js:50255:18
GatsbyNodeProvider@http://localhost:8005/component---src-templates-styleguide-jsx.js:101034:39
useObserver@http://localhost:8005/component---src-templates-styleguide-jsx.js:119470:64
StartSidecarNodes@http://localhost:8005/component---src-templates-styleguide-jsx.js:95889:23
StartSidecarNodes
StartSidecarNodes@http://localhost:8005/component---src-templates-styleguide-jsx.js:99788:34
StartSidecarNodes
StartSidecarNodes
StartSidecarNodes@http://localhost:8005/component---src-templates-styleguide-jsx.js:99956:74
StartSidecarNodes
StartSidecarNodes
StartSidecarNodes
StartSidecarNodes
PageRenderer@http://localhost:8005/commons.js:19472:34
PageQueryStore@http://localhost:8005/commons.js:20012:34
RouteHandler
div
FocusHandlerImpl@http://localhost:8005/commons.js:21854:20
FocusHandler@http://localhost:8005/commons.js:21825:18
RouterImpl@http://localhost:8005/commons.js:21730:20
Location@http://localhost:8005/commons.js:21559:22
Router
ScrollHandler@http://localhost:8005/commons.js:6627:40
RouteUpdates@http://localhost:8005/commons.js:19205:36
EnsureResources@http://localhost:8005/commons.js:9986:34
LocationHandler@http://localhost:8005/commons.js:20504:34
LocationProvider@http://localhost:8005/commons.js:21579:20
Location@http://localhost:8005/commons.js:21559:22
Root
dt@http://localhost:8005/commons.js:7890:12
StaticQueryStore@http://localhost:8005/commons.js:20100:36
ErrorBoundary@http://localhost:8005/commons.js:11265:40
DevOverlay@http://localhost:8005/commons.js:16656:22
RootWrappedWithOverlayAndProvider
App@[http://localhost:8005/commons.js:9194:74] 
 </code>

### PR DESCRIPTION
**Pull request by wodenx**
## Changes
Adds some basic list tokens and varitaions to flow container.

## Related Issues
Fixes #1559

### CODE REVIEW

File: packages/vital-list/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@bodiless/vital-list",
+  "version": "1.0.0-beta.4",
+  "description": "Editable Lists for Bodiless/VitalDS",
+  "author": "Chris Odem <coden@its.jnj.com>",
```
Comment (Guilherme-Almeida-Zeni) [inline packages/vital-list/package.json:5]: @wodenx ,

Minor comment here, it seems there is a typo in your name... it should be "Oden", right?

**Review by Guilherme-Almeida-Zeni — APPROVED** [review]

File: packages/vital-list/package.json (hunk 1 above)
Comment (wodenx) [inline packages/vital-list/package.json:5]:
```suggestion
  "author": "Chris Oden <coden@its.jnj.com>",
```

**Review by Guilherme-Almeida-Zeni — APPROVED** [review]

File: packages/vital-list/src/components/TitleInfographic/indexstatic.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
```
Comment (hvanyo) [inline packages/vital-list/src/components/TitleInfographic/indexstatic.ts:1]: @wodenx  this should be called index.static.ts

File: packages/vital-list/doc/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# VitalDS tokens for adding site metatags.
+
+- Provides main component as cxMeta
```
Comment (dewen) [inline packages/vital-list/doc/README.md:3]: cxMeta => vitalMeta

File: packages/vital-list/doc/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# VitalDS tokens for adding site metatags.
```
Comment (dewen) [inline packages/vital-list/doc/README.md:1]: replace with vital-list documentation

File: packages/vital-list/bodiless.docs.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+  "VitalDesignSystem": {
+    "Components": {
+      "SEO": {
+        "Metadata": {
+          "vitalMeta.md": "./doc/README.md"
```
Comment (dewen) [inline packages/vital-list/bodiless.docs.json:6]: vitalList.md

**Review by hvanyo — APPROVED** [review]

**Pull request merged**